### PR TITLE
Added immutable standard property

### PR DESCRIPTION
**Pull request by zonia3000**
I don't know if we should also add a sentence about returning a PermissionDenied fault if a user attempt to modify an immutable node.

### CODE REVIEW

File: VOSpace.tex
Comment (Zarquan) [inline VOSpace.tex]: Move the details of what `immutable` means to a separate section, giving us more room to describe it in detail.
```suggestion
    \item ivo://ivoa.net/vospace/core\#immutable MAY be used as the property URI denoting a node whose content cannot be changed and for which moveNode and deleteNode operations are not allowed. See section \ref{tbd} for details.
```

Comment (zonia3000) [reply VOSpace.tex]: I created a new section. I put it under "Nodes and node types" section, since it describes a special nodes behavior. Let me know if you prefer to move it somewhere else, for example at the end of the properties section. I added also a condition for link nodes (target can't be changed), but I'm not sure if it should be possible to change the target using the setNode operation.

File: VOSpace.tex
Comment (Zarquan) [inline VOSpace.tex:381]: Spaces in a label probably work, but why take the chance?
```suggestion
\label{subsubsec:immutable-nodes}
```

File: VOSpace.tex
Comment (Zarquan) [inline VOSpace.tex]: Just a suggestion.
```suggestion
A VOSpace service MAY support immutable nodes. Content of immutable nodes can't be changed and they are identified by adding the standard property ivo://ivoa.net/vospace/core\#immutable set to true.
```

File: VOSpace.tex
Comment (Zarquan) [reply VOSpace.tex]: Looks good. :+1:

Comment (Zarquan) [reply VOSpace.tex]: > I'm not sure if it should be possible to change the target using the setNode operation.

The target is effectively the content for a LinkNode, so it once the node is made immutable it you shouldn't be able to direct the link to point to something else.

File: VOSpace.tex
Comment (zonia3000) [reply VOSpace.tex:381]: I used the space because I noticed that it is used in other parts of the documents (e.g. `\label{subsec:nodes and node types}` and `\label{sec:vospace data model}`), but I agree that using a hyphen is better.

Comment (zonia3000) [reply VOSpace.tex:381]: It seems that all the labels of the document contain spaces.

Comment (Zarquan) [reply VOSpace.tex:381]: I hadn't seen that, serves me right for not looking. The references to section labels with spaces seem to work fine, so not an issue then.

File: VOSpace.tex
Comment (zonia3000) [reply VOSpace.tex]: Done.

**Comment by Zarquan** [comment]
Good to go. Asking @brianmajor, @gtaffoni or @cbanek to review and merge this.
(*) technically I know I do have write access to this repository, but only because I was involved in the transfer from Volute.

If we are following the proposed process for editing IVOA documents, then either the document editor or the chair or vice-chair of the working group should have the final say in accepting a PR.

**Review by brianmajor — APPROVED** [review]
Looks good.  I think I mentioned this before but we at CADC have an `isLocked` the serves the same purpose and follows the same rules you describe here.  `immutable` is a better term I think.  If we need to interoperate we can change the name of our property at that point.

If we add more detail about access control mechanisms in VOSpace perhaps then we could describe the circumstances in which nodes can be locked and unlocked.

**Pull request merged**
